c-data-channels/data-channels.c

```diff
@@ -0,0 +1,33 @@
+#include "webrtc.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+void on_open(GoDataChannel d)
+{
+    while (1)
+    {
+        GoSendText(d, "hello");
```

Comment:
Mind adding a simple C function to generate a pseudo-random string? Then just doing something like `fmt.Printf("Sending '%s'\n", message)`

Just put a bunch of acceptable characters in a char charset[] and then pick them out via rand()